src/EventStore.TestClient/ClientOptions.cs

```diff
@@ -56,6 +57,7 @@ public ClientOptions()
             WriteWindow = 2000;
             PingWindow = 2000;
             Force = false;
+            Reconnect = false;
```

Comment:
We need Reconnect to be true by default, and then to change it to false when we want to properly close the connection without reconnecting. We don't want to lose the reconnect behaviour while the test is running

So for example in the DvuBasicProcessor just before finishing Write():

```status.FinilizeStatus(writerIdx, failed != sent);
context.Client.Options.Reconnect = false;
connection.Close();
finish.Set();```